docs/pages/adopters.mdx

```diff
@@ -6,8 +6,9 @@ h1: Teleport Adopters and Use-Cases
 
 ## Who is using Teleport?
 
-Whether you are using Teleport OSS, Enteprise or Cloud, tell us about your use-case.
-Just click "Improve the docs" below and send us a PR.
+Whether you're using Teleport Comunity, Enteprise or Cloud, tell us about your use-case.
```

Comment:
Something like this will be more accurate: 

Whether you're using Teleport Open Source, Pro or Enterprise, self-hosted or in the cloud, tell us about your use-case